README.md

```diff
@@ -13,9 +13,13 @@
 * [OpenStack](docs/user/openstack/README.md)
 * [OpenStack (UPI)](docs/user/openstack/install_upi.md)
 * [Power](docs/user/power/install_upi.md)
+* [Power Virtual Server](https://docs.openshift.com/container-platform/latest/installing/installing_ibm_powervs/preparing-to-install-on-ibm-power-vs.html)
 * [vSphere](docs/user/vsphere/README.md)
 * [vSphere (UPI)](docs/user/vsphere/install_upi.md)
 * [z/VM](docs/user/zvm/install_upi.md)
+* [IBM Cloud](https://docs.openshift.com/container-platform/latest/installing/installing_ibm_cloud/preparing-to-install-on-ibm-cloud.html)
+* [Nutanix](https://docs.openshift.com/container-platform/latest/installing/installing_nutanix/preparing-to-install-on-nutanix.html)
```

Comment:
Can you put these two in alphabetical order

Reply:
yup, done :D